pkg/cloud/services/elb/loadbalancer.go

```diff
@@ -780,6 +782,15 @@ func (s *Service) reconcileELBTags(lb *infrav1.ClassicELB, desiredTags map[strin
 	return nil
 }
 
+func (s *Service) getClassicELBProtocol() *infrav1.ClassicELBProtocol {
```

Comment:
This protocol will only be used for health checks so should be named accordingly.

Reply:
Done